spec/spec_helper.rb

```diff
@@ -37,7 +37,7 @@ def stub_github_get_pullrequests(repo_name, branch_name)
       .to_return(
         status: 200,
         headers: { "Content-Type": "application/json" },
-        body: [].to_json,
+        body: [{}].to_json,
```

Comment:
Oh I see - I'm guessing your first commit doesn't work, and your second commit is what fixes it. Let's combine the two commits together (every commit should be a complete, self-contained change, with a fully passing test suite).